src/index.js

```diff
@@ -1,31 +1,16 @@
 import './style.css';
 
-import {
-  createNewTask,
-  deleteTask,
-  loadTasks,
-  titleElement,
-} from './modules/tasks.js';
+import { initializeTasks, createNewTask } from './modules/tasks.js';
 
 const input = document.querySelector('#new-task-input');
 const submitButton = document.querySelector('.add');
-const taskList = document.querySelector('#tasks');
-loadTasks();
 
-// Add task
+initializeTasks();
+
 submitButton.onclick = (e) => {
   e.preventDefault();
   if (input.value !== '') {
     createNewTask(input.value);
     input.value = '';
   }
 };
```

Comment:
**[OPTIONAL]**

- [ ] It would be nice if you make your  commit messages capitalized to make your pull request professional :+1: